website/content/docs/k8s/installation/deployment-configurations/clients-outside-kubernetes.mdx

```diff
@@ -35,14 +48,69 @@ annotation `consul.hashicorp.com/auto-join-port` to an integer value or
 
-Consul typically requires a fully connected network.
-Because the Consul Helm chart currently doesn't allow exposing servers' gossip ports via a `hostPort`,
-nodes outside of Kubernetes joining a cluster running within Kubernetes must be able to communicate
-to pod IPs via the network. Note that the auto-join provider discussed above will use pod IPs by default.
+2. Make sure you have the `kubeconfig` file for the Kubernetes cluster in `$HOME/.kube/config`.
+
+2.  On the external VM, run: 
+```shell-session
+consul agent \
+  -advertise="$ADVERTISE_IP" \
+  -retry-join='provider=k8s label_selector="app=consul,component=server"'
+  -bind=0.0.0.0 \
+  -client=0.0.0.0 \
+  -hcl='leave_on_terminate = true' \
+  -hcl='ports { grpc = 8502 }' \
+  -config-dir=$CONFIG_DIR \
+  -datacenter=$DATACENTER \
+  -data-dir=$DATA_DIR \
+```
+
+3. Check if the join was successful by running `consul members`
+
+### Auto-join on the Host network
+If you don't have connectivity to the `podIP:containerPort` for clients and servers deployed in Kubernetes, you have the option to expose the clients and servers on the host network. 
+
+1. Install the [official Helm chart](/docs/k8s/helm) with the following values:
+```yaml
+client:
+  exposeGossipPorts: true # exposes client gossip ports as hostPorts
+server:
+  exposeGossipAndRPCPorts: true # exposes the server gossip and RPC ports as hostPorts
+  ports:
+    # Configures the server gossip port
+    serflan:
+      # Note that this needs to be different than 8301, to avoid conflicting with the client gossip hostPort
+      port: 9301 
+```
+ This will expose the client gossip ports, the server gossip ports and the server RPC port at `hostIP:hostPort`. Note that `hostIP` is the **internal** IP of the VM that the client/server pods are deployed on.
+
+2. Make sure the pod IPs of the clients and servers in Kubernetes are routeable from the VM and that the VM IP (`$ADVERTISE_IP`) is routeable from the pods, adding any firewall rules necessary to allow traffic on the gossip and RPC ports for the K8s Nodes and the external VM.
+
+3. Make sure you have the `kubeconfig` file for the Kubernetes cluster in `$HOME/.kube/config`.
+
+4. On the external VM, run (note the addition of `host_network=true` in the retry-join argument): 
+```shell-session
+consul agent \
+  -advertise="$ADVERTISE_IP" \
+  -retry-join='provider=k8s host_network=true label_selector="app=consul,component=server"'
+  -bind=0.0.0.0 \
+  -client=0.0.0.0 \
+  -hcl='leave_on_terminate = true' \
+  -hcl='ports { grpc = 8502 }' \
+  -config-dir=$CONFIG_DIR \
+  -datacenter=$DATACENTER \
+  -data-dir=$DATA_DIR \
+``` 
+3. Check if the join was successful by running `consul members`
 
--> **Consul Enterprise customers** may use
-[network segments](/docs/enterprise/network-segments) to
-enable non-fully-connected topologies. However, out-of-cluster nodes must still
-be able to communicate with the server pod or host IP addresses.
+## Manual join
```

Comment:
should we recommend to use the host port option for the manual join since the pod IPs could change during upgrades?

Reply:
Yeah maybe instead let's recommend setting up a DNS entry that would resolve to the server's pod IPs (if using pod network) or node ips that the server pods are running on (if using host ports)?

Reply:
Yes that makes sense, thank you both. I've included that. Also, it doesn't need every consul server, right? I'm going to take out that phrase and say "pass the address of at least one consul server".